acme/acme/client_test.py

```diff
@@ -432,9 +449,22 @@ def test_revoke_bad_status_raises_error(self):
             self.certr,
             self.rsn)
 
+class MockJSONDeSerializable(jose.JSONDeSerializable):
+    # pylint: disable=missing-docstring
+    def __init__(self, value):
+        self.value = value
+
+    def to_partial_json(self):
+        return {'foo': self.value}
+
+    @classmethod
+    def from_json(cls, value):
+        pass  # pragma: no cover
+
 
 class ClientNetworkTest(unittest.TestCase):
     """Tests for acme.client.ClientNetwork."""
+    # pylint: disable=too-many-public-methods
```

Comment:
I don't think this comment is needed.

Reply:
Pylint errors without it:

```
$ pylint --reports=n --rcfile=.pylintrc acme/acme/client_test.py
************* Module acme.client_test
R:465, 0: Too many public methods (21/20) (too-many-public-methods)
```

Reply:
Huh. I certainly tried to test this before I added this comment, but obviously I didn't do so correctly. Sorry for the misleading noise!